source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -9,21 +9,21 @@ Formats for COUNTER Reports
 
 R5 reports can be delivered in tabular form, or as machine-readable data (JSON) via the COUNTER_SUSHI API. The tabular form MUST be provided as either an Excel or a tab-separated-value (TSV) file, or both. Additional file formats that can be easily imported into spreadsheet programs without loss or corruption may be offered at the vendor's discretion. The reports in JSON, TSV and other text formats MUST be encoded using UTF-8. The JSON format MUST comply with the COUNTER_SUSHI API Specification (see :numref:`sushi` below).
 
-All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the “Journal Requests (Controlled)” Standard View. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
+All COUNTER reports have the same layout and structure. Figure 3.b (below) provides an example of the Title Report. Figure 3.c (below) shows the layout for tabular reports, which will be the focus of the discussions throughout this document. Note that the COUNTER_SUSHI API Specification includes the same elements with the same or similar names; therefore, understanding the tabular reports translates to an understanding of what is REQUIRED in reports retrieved via the COUNTER_SUSHI API.
 
-.. figure:: ../_static/img/3b-standard-view-sample.png
-   :alt: Standard View sample
+.. figure:: ../_static/img/Figure-3b.png
```

Comment:
The Title Report needs to be scaled down a lot to fit on the page and is therefore difficult to read. I would suggest to only use the left half of the current images (with the fixes below).

The Release must be formatted as string. The Metric_Types filter is the default, therefore the list must be omitted. Attributes_To_Show is missing.